code/modules/jobs/jobs.dm

```diff
@@ -304,9 +304,9 @@ GLOBAL_LIST_INIT(exp_jobsmap, list(
 	EXP_TYPE_SILICON = list("titles" = list("AI","Cyborg")),
 	EXP_TYPE_SERVICE = list("titles" = civilian_positions),
 
-	EXP_TYPE_FALLOUT = list("titles" = brotherhood_positions | oasis_positions | legion_positions | ncr_positions | vault_positions | wasteland_positions | tribal_positions | followers_positions | enclave_positions),
```

Comment:
Don't touch this.
This counts up EXP for the sake of a general timelock. If you remove BOS from here it's *just* a pain in the ass.
If you keep it in, Nothing bad happens.

Reply:
That makes sense, I was just removing everything I thought would mess with the job select screen, but I should have read the variable name.

Reply:
Job select screen will be set in a fix PR coming up later tnight.
That's actually part of where I specialize.